cryptostore/data/influx.py

```diff
@@ -92,6 +92,10 @@ def write(self, exchange, data_type, pair, timestamp):
             for entry in self.data:
                 ts = int(Decimal(entry["timestamp"]) * 1000000000)
                 agg.append(f'{data_type}-{exchange},pair={pair},exchange={exchange} open_interest={entry["open_interest"]},timestamp={entry["timestamp"]},receipt_timestamp={entry["receipt_timestamp"]} {ts}')
+        elif data_type == LIQUIDATIONS:
+            for entry in self.data:
+                ts = int(Decimal(entry["timestamp"]) * 1000000000)
+                agg.append(f'{data_type}-{exchange},pair={pair},exchange={exchange} side="{entry["side"]}",leaves_qty={entry["leaves_qty"]},order_id="{entry["order_id"]}",price="{entry["price"]}",times$
```

Comment:
looks like a bug at the end of this line - was this tested?

Reply:
Yes, bad copy/paste , update : 


```python
elif data_type == LIQUIDATIONS:
            for entry in self.data:
                ts = int(Decimal(entry["timestamp"]) * 1000000000)
                agg.append(f'{data_type}-{exchange},pair={pair},exchange={exchange} side="{entry["side"]}",leaves_qty={entry["leaves_qty"]},order_id="{entry["order_id"]}",price="{entry["price"]}",timestamp={entry["timestamp"]},receipt_timestamp={entry["receipt_timestamp"]} {ts}')

```

Reply:
Tested and worked actually on my server